pact/provider/assertj/src/main/java/org/arquillian/algeron/pact/provider/assertj/ProviderUrlAssert.java

```diff
@@ -0,0 +1,19 @@
+package org.arquillian.algeron.pact.provider.assertj;
+
+import java.net.URL;
+
+public class ProviderUrlAssert {
```

Comment:
The tricky part is that by using this construct you won't be able to make call to other methods from the parent assertion class unless you do something like `.up()`,  next to holding reference to the owning object. 

I would still consider flating it. We don't assert anything about the URL anyway, it's just a configuration item in this context.